modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/AccountGroupResourceTest.java

```diff
@@ -313,6 +328,16 @@ private AccountGroup _postAccountGroup(AccountGroup accountGroup)
 		return accountGroupResource.postAccountGroup(accountGroup);
 	}
 
+	private AccountEntry _randomAccountEntry() throws Exception {
```

Comment:
sorry @matyaswollner could you please rename this to `_ addAccountEntry`? To be equal to the method we are calling